proto/proto-backplane-grpc/src/main/proto/deephaven/proto/config.proto

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+syntax = "proto3";
+
+package io.deephaven.proto.backplane.grpc;
+
+option java_multiple_files = true;
+option optimize_for = SPEED;
+option go_package = "github.com/deephaven/deephaven-core/go/internal/proto/config";
+
+/**
+ * Provides simple configuration data to users. Unauthenticated users may call GetAuthenticationConstants
+ * to discover hints on how they should proceed with providing their identity, while already-authenticated
+ * clients may call GetConfigurationConstants for details on using the platform.
+ */
+service ConfigService {
+  rpc GetAuthenticationConstants(AuthenticationConstantsRequest) returns (AuthenticationConstantsResponse) {}
+  rpc GetConfigurationConstants(ConfigurationConstantsRequest) returns (ConfigurationConstantsResponse) {}
+}
+
+message AuthenticationConstantsRequest {}
+message ConfigurationConstantsRequest {}
+message AuthenticationConstantsResponse {
+  map<string, ConfigValue> config_values = 1;
+}
+message ConfigurationConstantsResponse {
+  map<string, ConfigValue> config_values = 1;
+}
+
+message ConfigValue {
+  oneof kind {//leave room for more types
+    string string_value = 2;
```

Comment:
Please set as value 3 to match struct.proto.

Reply:
Ill make the change for easy compat, but I wouldn't consider struct.proto to be a goal or even good code. It "works" as a "well I guess most things can interchange with this if they can at least interop with JSON cleanly", but you effectively have to convert to JSON to be able to query it cleanly, instead of knowing what types/keys to expect, at least not without some irritating boilerplate ("Key `xyz` is a list of strings, let's look for it. Is it present? Okay good, read the Value. Is it a null? No, okay good, we don't return a null value. Is it a ListValue? Okay, data isn't mangled. Read each Value..."). Additionally, despite JSON specifying no limits to math precision, but allowing that implementations might support some limits, Value allows `double`, so int32 values have to be narrowed when read from a strongly typed language, and int64 values may lose precision (or get encoded as strings...).

I'd nearly rather have CSV.